src/app/messages/createIntl.ts

```diff
@@ -0,0 +1,24 @@
+import memoize from 'lodash/fp/memoize';
+import { createIntl, createIntlCache } from 'react-intl';
+import Sentry from '../../helpers/Sentry';
+import enMessages from '../messages/en/index';
```

Comment:
you're importing the english messages unconditionally here, you need to defer this to the catch and if that also fails just explode everywhere